SIT.Manager/ViewModels/MainViewModel.cs

```diff
@@ -70,7 +73,14 @@ public partial class MainViewModel : ObservableRecipient, IRecipient<Installatio
 
         CloseButtonCommand = new RelayCommand(() => { UpdateAvailable = false; });
 
-        _managerConfigService.ConfigChanged += async (o, c) => await CheckForUpdate();
+        _managerConfigService.ConfigChanged += ManagerConfigService_ConfigChanged;
+    }
+
+    private async void ManagerConfigService_ConfigChanged(object? sender, ManagerConfig e)
+    {
+        IsDevloperModeEnabled = e.EnableDeveloperMode;
+
+        await CheckForUpdate();
     }
```

Comment:
Maybe silly question. Why do we check for an update when toggling dev mode and if this is needed will caching get in the way of this working properly?

Reply:
That's been there the whole time, I just stuck ```IsDevloperModeEnabled = e.EnableDeveloperMode;``` in there as that was the most appropriate place for it.

probably should look into that separately as it does currently run the app update check every time that there is a ManagerConfig change. It does also check for a SIT update but that only happens once every hour